srcpkgs/glib/template

```diff
@@ -16,7 +16,7 @@ license="LGPL-2.1-or-later"
 homepage="https://wiki.gnome.org/Projects/GLib"
 changelog="https://gitlab.gnome.org/GNOME/glib/raw/master/NEWS"
```

Comment:
Changelog has moved to https://gitlab.gnome.org/GNOME/glib/raw/main/NEWS but the relevant one for this version is https://gitlab.gnome.org/GNOME/glib/raw/glib-2-68/NEWS

Reply:
Ah, nice! Thanks!